common/lib/xmodule/xmodule/video_module/video_module.py

```diff
@@ -711,19 +714,21 @@ def definition_to_xml(self, resource_fs):
                 ele.set('src', self.transcripts[transcript_language])
                 xml.append(ele)
 
-        if edxval_api:
-            external, video_ids = get_video_ids_info(self.edx_video_id, self.youtube_id_1_0, self.html5_sources)
-            if video_ids:
-                try:
-                    xml.append(
-                        edxval_api.export_to_xml(
-                            video_ids,
-                            unicode(self.runtime.course_id.for_branch(None)),
-                            external=external
-                        )
+        if self.edx_video_id and edxval_api:
```

Comment:
As `edxval` is a must have dependency for platform so second part of the above condition is unnecessary.